README.md

```diff
@@ -55,6 +55,7 @@ systemusersAPI = jcapiv1.SystemusersApi()
 try:
     # make an API call to retrieve all systemusers:
     users = systemusersAPI.systemusers_list(content_type, accept)
+    print users
```

Comment:
Elsewhere the code is using a `print("message")` format but here we are not, are examples generated as well?

Reply:
Those are just my own examples. I just added the print statement cause I realized I wasn't doing anything with the result so I might as well print it.